README.md

```diff
@@ -30,6 +30,7 @@ Breathing gymnastics is a system of breathing exercises that focuses on the trea
 Nafas is a collection of breathing gymnastics designed to reduce the exhaustion of long working hours.
 With multiple breathing patterns, Nafas helps you find your way to a detoxified energetic workday and also improves your concentration by increasing the oxygen level.
 No need to walk away to take a break, just sit comfortably, run Nafas and let the journey begin.
+In Persian, **Nafas** means breath.
```

Comment:
do we need `,` here? if yes can't we changed to `**Nafas** means breath in Persian.`

Reply:
solved in https://github.com/sepandhaghighi/nafas/pull/20/commits/f8884b665b39612cad644ab8ce0ae8f9d2e26a4b 👍